hazelcast/src/main/java/com/hazelcast/durableexecutor/impl/DurableExecutorContainer.java

```diff
@@ -138,6 +166,9 @@ public void run() {
             } finally {
                 if (!isCancelled()) {
                     setResponse(response);
+                    if (statisticsEnabled) {
```

Comment:
Does the "execution time" include the queueing time in all of the uses of the executor stats?

Reply:
No, it's meaning is aligned with executor service and it is the execution time in thread without queueing time.